src/kernels/jupyter/launcher/serverUriStorage.ts

```diff
@@ -226,22 +226,22 @@ export class JupyterServerUriStorage implements IJupyterServerUriStorage, IServe
     }
     public async setUriToLocal(): Promise<void> {
         traceInfoIfCI(`setUriToLocal`);
-        await this.setUri(Settings.JupyterServerLocalLaunch);
+        await this.setUri(Settings.JupyterServerLocalLaunch, undefined);
     }
     public async setUriToRemote(uri: string, displayName: string): Promise<void> {
         // Make sure to add to the saved list before we set the uri. Otherwise
         // handlers for the URI changing will use the saved list to make sure the
         // server id matches
         await this.addToUriList(uri, Date.now(), displayName);
-        await this.setUri(uri);
+        await this.setUri(uri, displayName);
     }
 
     public async setUriToNone(): Promise<void> {
         traceInfoIfCI(`setUriToNone`);
-        return this.setUri(undefined);
+        return this.setUri(undefined, undefined);
     }
 
-    public async setUri(uri: string | undefined) {
+    public async setUri(uri: string | undefined, disaplayName: string | undefined) {
```

Comment:
Could you fix the misspell here for display? Aside from that looks great.

Reply:
oh where is my spell checker ;(